README-CLI-CLOUD.md

```diff
@@ -156,6 +162,18 @@ Options:
 | `--thumbprint`     | Use to specify the SHA1 thumbprint of the certificate to renew. Value may be specified as a string or read from the certificate file using the `file:` prefix. |
 
 
+
+## Certificate Retire Parameters
+```
+vcert retire -k <api key> [--id <request id> | --thumbprint <sha1 thumb>]
+```
+Options:
+
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Command&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description                                                  |
+| -------------- | ------------------------------------------------------------ |
+| `--id`         | Use to specify the unique identifier of the certificate to revoke.  Value may be specified as a string or read from a file using the `file:` prefix. |

```

Comment:
Here it should be `retire` instead of `revoke`.

Reply:
fixed in new commit